### PR TITLE
Delete riscv from riscv64

### PR DESCRIPTION
**Pull request by luyahan**

### CODE REVIEW

**Review by fw-brice — CHANGES_REQUESTED** [review]
I suppose we should also remove all of the uses of `V8_TARGET_ARCH_RISCV` as well.

**Comment by fw-brice** [comment]
Please also review the formatting issue reported by the precheck job.

File: tools/testrunner/local/statusfile.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
 for var in ["debug", "release", "big", "little", "android",
             "arm", "arm64", "ia32", "mips", "mipsel", "mips64", "mips64el",
             "x64", "ppc", "ppc64", "s390", "s390x", "macos", "windows",
-            "linux", "aix", "r1", "r2", "r3", "r5", "r6", "riscv", "riscv64"]:
+            "linux", "aix", "r1", "r2", "r3", "r5", "r6", "riscv","riscv64"]:
```
Comment (fw-brice) [inline tools/testrunner/local/statusfile.py:66]: `"riscv"` is still in this list.

File: tools/testrunner/local/statusfile.py [hunk 2]
```diff
@@ -77,6 +77,7 @@ def __init__(self, path, variables):
     _prefix_rules: {variant: {test name prefix: [rule]}}
     """
     self.variables = variables
+    print(path)
```
Comment (fw-brice) [inline tools/testrunner/local/statusfile.py:80]: Remove this debug print.

**Review by fw-brice — CHANGES_REQUESTED** [review]
There are still many uses of `V8_TARGET_ARCH_RISCV`:

```
$ grep -rw V8_TARGET_ARCH_RISCV *
src/interpreter/interpreter-assembler.cc:    V8_TARGET_ARCH_RISCV
src/snapshot/deserializer.h:    defined(V8_TARGET_ARCH_RISCV) || V8_EMBEDDED_CONSTANT_POOL
src/regexp/regexp-macro-assembler-arch.h:#elif V8_TARGET_ARCH_RISCV
src/regexp/regexp.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/wasm/baseline/liftoff-assembler.h:#elif V8_TARGET_ARCH_RISCV
src/wasm/baseline/liftoff-assembler-defs.h:#elif V8_TARGET_ARCH_RISCV
src/wasm/wasm-linkage.h:#elif V8_TARGET_ARCH_RISCV
src/runtime/runtime-atomics.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/runtime/runtime-atomics.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/objects/code.h:#elif V8_TARGET_ARCH_RISCV
src/objects/backing-store.cc:#elif V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins.cc:    V8_TARGET_ARCH_RISCV
src/builtins/builtins.cc:        // V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/reloc-info.cc:    defined(V8_TARGET_ARCH_RISCV64) || defined(V8_TARGET_ARCH_RISCV)
src/codegen/register-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/constants-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/external-reference.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/macro-assembler.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/assembler-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/assembler-inl.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/register-configuration.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/interface-descriptors.cc:    !defined(V8_TARGET_ARCH_RISCV64) && !defined(V8_TARGET_ARCH_RISCV)
src/execution/simulator.h:#elif V8_TARGET_ARCH_RISCV
src/execution/frame-constants.h:#elif V8_TARGET_ARCH_RISCV
src/diagnostics/perf-jit.h:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/profiler/tick-sample.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-selector.cc:    !V8_TARGET_ARCH_RISCV64 && !V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-selector.cc:        // !V8_TARGET_ARCH_RISCV64 && !V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-codes.h:#elif V8_TARGET_ARCH_RISCV
src/logging/log.cc:#elif V8_TARGET_ARCH_RISCV
src/base/platform/platform-posix.cc:#elif V8_TARGET_ARCH_RISCV
src/base/compiler-specific.h:      !defined(V8_TARGET_ARCH_RISCV64) && !defined(V8_TARGET_ARCH_RISCV)) || \
src/debug/debug-evaluate.cc:    defined(V8_TARGET_ARCH_RISCV)
test/cctest/test-regexp.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV grep -rw V8_TARGET_ARCH_RISCV *                   
src/interpreter/interpreter-assembler.cc:    V8_TARGET_ARCH_RISCV
src/snapshot/deserializer.h:    defined(V8_TARGET_ARCH_RISCV) || V8_EMBEDDED_CONSTANT_POOL
src/regexp/regexp-macro-assembler-arch.h:#elif V8_TARGET_ARCH_RISCV
src/regexp/regexp.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/wasm/baseline/liftoff-assembler.h:#elif V8_TARGET_ARCH_RISCV
src/wasm/baseline/liftoff-assembler-defs.h:#elif V8_TARGET_ARCH_RISCV
src/wasm/wasm-linkage.h:#elif V8_TARGET_ARCH_RISCV
src/runtime/runtime-atomics.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/runtime/runtime-atomics.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/objects/code.h:#elif V8_TARGET_ARCH_RISCV
src/objects/backing-store.cc:#elif V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:    V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins-sharedarraybuffer-gen.cc:        // || V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/builtins/builtins.cc:    V8_TARGET_ARCH_RISCV
src/builtins/builtins.cc:        // V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/reloc-info.cc:    defined(V8_TARGET_ARCH_RISCV64) || defined(V8_TARGET_ARCH_RISCV)
src/codegen/register-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/constants-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/external-reference.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/macro-assembler.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/assembler-arch.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/assembler-inl.h:#elif V8_TARGET_ARCH_RISCV
src/codegen/register-configuration.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/codegen/interface-descriptors.cc:    !defined(V8_TARGET_ARCH_RISCV64) && !defined(V8_TARGET_ARCH_RISCV)
src/execution/simulator.h:#elif V8_TARGET_ARCH_RISCV
src/execution/frame-constants.h:#elif V8_TARGET_ARCH_RISCV
src/diagnostics/perf-jit.h:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/profiler/tick-sample.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-selector.cc:    !V8_TARGET_ARCH_RISCV64 && !V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-selector.cc:        // !V8_TARGET_ARCH_RISCV64 && !V8_TARGET_ARCH_RISCV
src/compiler/backend/instruction-codes.h:#elif V8_TARGET_ARCH_RISCV
src/logging/log.cc:#elif V8_TARGET_ARCH_RISCV
src/base/platform/platform-posix.cc:#elif V8_TARGET_ARCH_RISCV
src/base/compiler-specific.h:      !defined(V8_TARGET_ARCH_RISCV64) && !defined(V8_TARGET_ARCH_RISCV)) || \
src/debug/debug-evaluate.cc:    defined(V8_TARGET_ARCH_RISCV)
test/cctest/test-regexp.cc:#elif V8_TARGET_ARCH_RISCV64 || V8_TARGET_ARCH_RISCV
```

File: tools/testrunner/local/statusfile.py (hunk 1 above)
Comment (luyahan) [reply tools/testrunner/local/statusfile.py]: delete it needs delete 'riscv' from *.status

Comment (luyahan) [reply tools/testrunner/local/statusfile.py]: I had do it

File: tools/testrunner/local/statusfile.py (hunk 2 above)
Comment (luyahan) [reply tools/testrunner/local/statusfile.py]: I fix it

**Review by fw-brice — CHANGES_REQUESTED** [review]
Looks good, I just found one remaining place with the RISCV macro:
test/cctest/test-regexp.cc:615

**Review by fw-brice — APPROVED** [review]
LGTM

**Pull request merged**